packages/ui/src/lib/components/discord/MessageAuthor.tsx

```diff
@@ -1,11 +1,16 @@
 export interface IDiscordMessageAuthor {
 	avatar: string;
-	bot?: boolean;
+	bot?:
+		| boolean
+		| {
+				verified: boolean;
+		  };
```

Comment:
I'd love it if this were just two properties instead, both optional. `bot` and `verified`. Standalone verified would just do nothing, and with bot being true, it would add the check.

Seems a lot more straightforward than having two vastly different interfaces to provide.